plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/StatsResetCommand.java

```diff
@@ -0,0 +1,48 @@
+package net.dzikoysk.funnyguilds.feature.command.user;
+
+import java.util.List;
+import net.dzikoysk.funnycommands.stereotypes.FunnyCommand;
+import net.dzikoysk.funnycommands.stereotypes.FunnyComponent;
+import net.dzikoysk.funnyguilds.feature.command.AbstractFunnyCommand;
+import net.dzikoysk.funnyguilds.shared.FunnyFormatter;
+import net.dzikoysk.funnyguilds.shared.bukkit.ItemUtils;
+import net.dzikoysk.funnyguilds.user.User;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+@FunnyComponent
+public final class StatsResetCommand extends AbstractFunnyCommand {
+
+    @FunnyCommand(
+            name = "${user.stats-reset.name}",
+            description = "${user.stats-reset.description}",
+            aliases = "${user.stats-reset.aliases}",
+            permission = "funnyguilds.statsreset",
+            acceptsExceeded = true,
+            playerOnly = true
+    )
+    public void execute(Player player, User user) {
+        List<ItemStack> requiredItems = this.config.statsResetItems;
+        if (!ItemUtils.playerHasEnoughItems(player, requiredItems, this.messages.statsResetItems)) {
+            return;
+        }
+        int lastRank = user.getRank().getPoints();
```

Comment:
Przypisz sobie to user.getRank() do zmiennej np. po prostu `rank` i używaj tej zmiennej